spec/features/homepage/user_visits_homepage_spec.rb

```diff
@@ -1,33 +0,0 @@
-require 'rails_helper'
```

Comment:
This test bombs out 1/5 times. Let's remove it and use controller tests instead. There is a model test for the react component.